test/cookbooks/osl-git-test/metadata.rb

```diff
@@ -7,6 +7,4 @@
 description      'Installs/Configures osl-git-test'
 version          '0.1.0'
 
-depends          'osl-git'
-
-supports         'centos', '~> 7.0'
+depends          'osl-git'
```

Comment:
Please fix the newline on this file.

Reply:
No I meant this line is actually missing a newline (see the red icon). It should have one here.

Reply:
sorry, updated